sstables/writer_impl.hh

```diff
@@ -32,12 +33,14 @@ struct sstable_writer::writer_impl {
     const schema& _schema;
     const io_priority_class& _pc;
     const sstable_writer_config _cfg;
+    position_in_partition_tracker _pip_tracker;
 
     writer_impl(sstable& sst, const schema& schema, const io_priority_class& pc, const sstable_writer_config& cfg)
         : _sst(sst)
         , _schema(schema)
         , _pc(pc)
         , _cfg(cfg)
+        , _pip_tracker()
```

Comment:
Not needed, the compiler will call this automatically.

Reply:
It's a placeholder for the follow up patchset :)
But you're right, it could be dropped here and added later when needed.

Reply:
Dropped in v2.